src/backend/routers/iep.test.ts

```diff
@@ -32,4 +42,13 @@ test("basic flow - add/get goals and subgoals", async (t) => {
     goal_id: goal1!.goal_id,
   });
   t.is(gotSubgoals.length, 2);
+
+  t.truthy(
```

Comment:
🐧 Ideally, we should be using our API methods instead of directly querying the db. We will likely need a `getTasks` method down the line when we want to do the Para schedule view. You can add a TODO for that now.